plugins/LayerView/LayerViewProxy.py

```diff
@@ -36,6 +37,12 @@ def currentLayer(self):
         if type(active_view) == LayerView.LayerView.LayerView:
             return active_view.getCurrentLayer()
 
+    @pyqtProperty(int, notify = currentLayerChanged)
+    def minimumLayer(self):
+        active_view = self._controller.getActiveView()
+        if type(active_view) == LayerView.LayerView.LayerView:
```

Comment:
Why not ductype it? If a view has a getMinimumLayer() function, it is good enough. It shouldn't matter if it is a layerview or not.